ggml/src/ggml-sycl/im2col.cpp

```diff
@@ -1,9 +1,6 @@
-//
 // MIT license
 // Copyright (C) 2024 Intel Corporation
 // SPDX-License-Identifier: MIT
-//
-
```

Comment:
Was that change done on purpose? The same pattern is used in many places and looks fine to me.

Reply:
This format should not be changed.
It's standard copyright format of Intel for open source code contribution.

Reply:
Sorry about that. Restored.